[DOC-2869] Update images (#1166)
harness/developer-hub: docs/chaos-engineering/technical-reference/architecture.md

## Harness control plane

> **Harness control plane** is the single source to collaboratively create, schedule, and monitor chaos experiments. It comes with a set of chaos faults that are defined in a sequence, which helps achieve the desired chaos impact on the target resources. You can sign in to the Harness platform and leverage the interactive chaos infrastructure to define your chaos experiments and target the infrastructure. You can also monitor the experiments during their execution.

Reviewer: ```suggestion
**Harness control plane** is the single source to collaboratively create, schedule, and monitor chaos experiments. It comes with a set of chaos faults that are defined in a sequence, which helps achieve the desired chaos impact on the target resources. You can sign in to the Harness platform and leverage the interactive UI dashboard to enable chaos access in your infrastructure, define your chaos experiments and target the infrastructure. You can also monitor the experiments during their execution.
```

Author: We don't use 'Chaos Access' anymore, no? @neelanjan00

Reviewer: chaos access is merely a verb, we can use it as such. We are only prohibiting the use of chaos access as a noun. But we can use what you suggested as well.

> #### Authentication server

Reviewer: Auth server is no more there, we just use Harness auth now. We can remove this entirely.

The authentication server is a Golang microservice responsible for authorizing and authenticating the requests received from chaos infrastructure. It also helps manage users and their projects. It helps in user and project creation, user login, password resets, updating of user information, and managing project-related operations.

> #### Backend server

Reviewer: ```suggestion
#### Chaos manager
```
Backend server is called chaos manager now

> The backend server is a GraphQL-based Golang microservice that serves the requests received from the chaos infrastructure either by querying the database for relevant information or fetching information from the execution plane.

Reviewer: ```suggestion
The chaos manager is a GraphQL-based Golang microservice that serves the requests either by querying the database for relevant information or fetching information from the execution plane.
```
